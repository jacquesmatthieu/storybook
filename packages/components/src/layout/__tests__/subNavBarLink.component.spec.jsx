import React from 'react';
import { shallow } from 'enzyme';

import SubNavBarLink from '../subNavBarLink.component';

describe('SubNavBarLink', () => {
  let props;

  beforeEach(() => {
    props = {
      to: 'bar',
    };
  });

  const getWrapper = () => shallow(<SubNavBarLink {...props}>Foo</SubNavBarLink>);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
