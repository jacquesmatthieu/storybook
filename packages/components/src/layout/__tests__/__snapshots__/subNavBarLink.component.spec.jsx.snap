// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubNavBarLink should render properly 1`] = `
<NavLink
  component={[Function]}
  data-test-id=""
  exact={false}
  location={
    Object {
      "pathname": "foo",
    }
  }
  match={
    Object {
      "params": Object {},
    }
  }
  params={Object {}}
  to="bar"
>
  <typographycomponent__Typography
    variant="header"
  >
    Foo
  </typographycomponent__Typography>
</NavLink>
`;
